docs/workforce-management-framework/topics/con-security-introduction.adoc

```diff
@@ -0,0 +1,49 @@
+[]
+= Introduction to {WFM-RC-NameShort} security features
+
+{WFM-RC-NameShort} server gives developers a choice one of the following authentication and authorisation providers
```

Comment:
Server? - Might cause confusion as the Auth provider is chosen in client apps too and not enforced by the server. Maybe "Demo solution" might be a better description.

"...a choice one of the following authentication and authorisation providers`:`..."

Reply:
have to avoid the use of demo as its not a demo going forward.
change to cloud server